Makefile.am

```diff
@@ -270,6 +276,8 @@ EXTRA_DIST = \
 	$(cc_proxy_sources) \
 	$(systemdservice_in_files) \
 	data/cc-oci-runtime.sh.in \
+	data/dockerFiles/Dockerfile.fedora.in \
+	data/dockerFiles/Dockerfile.ubuntu.in \
```

Comment:
Should I add it? there are no substitutions in Dockerfile.clearlinux as we work with latest clearlinux release